cmd/deploy/deploy.go

```diff
@@ -284,9 +284,11 @@ func (dc *DeployCommand) RunDeploy(ctx context.Context, deployOptions *Options)
 	oktetoLog.Debug("found okteto manifest")
 	dc.PipelineType = deployOptions.Manifest.Type
 
-	if deployOptions.Manifest.Deploy == nil && deployOptions.Manifest.Dependencies == nil {
+	dependenciesEmpty := deployOptions.Manifest.Dependencies == nil || len(deployOptions.Manifest.Dependencies) == 0
```

Comment:
when `.Dependencies` is an empty array (not `nil`), it would skip this condition instead of stopping here

Reply:
NIT: I'd change the variable name to `isDependenciesEmpty` or extract `deployOptions.Manifest.Dependencies` and use it directly. Looking at the variable name I don't know if it's a boolean or not

Reply:
NIT:NIT: `areDependenciesEmpty`

Reply:
also, maybe a manifest method attached to the struct? so `manifest.HasDependencies()` ? same for deploy section

Reply:
good idea @teresaromero 👍